addons/cargo/functions/fnc_paradropItem.sqf

```diff
@@ -0,0 +1,85 @@
+/*
+ * Author: marc_book, commy2, CAA-Picard
+ * Unload and paradrop object from plane or helicopter.
+ *
+ * Arguments:
+ * 0: Object <OBJECT>
+ * 1: Vehicle <OBJECT>
+ *
+ * Return value:
+ * Object unloaded <BOOL>
+ *
+ * Example:
+ * [object, vehicle] call ace_cargo_fnc_paradropItem
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_item", "_vehicle"];
+
+private "_loaded";
+_loaded = _vehicle getVariable [QGVAR(loaded), []];
+
+if !(_item in _loaded) exitWith {false};
+
+// unload item from cargo
+_loaded deleteAt (_loaded find _item);
+_vehicle setVariable [QGVAR(loaded), _loaded, true];
+
+private ["_cargoSpace", "_itemSize"];
+
+_cargoSpace = [_vehicle] call FUNC(getCargoSpaceLeft);
+_itemSize = [_item] call FUNC(getSizeItem);
+_vehicle setVariable [QGVAR(space), _cargoSpace - _itemSize, true];
+
+detach _item;
+
+private "_position";
+_position = (vectorDir _vehicle vectorMultiply -15) vectorAdd getPosASL _vehicle;
+
+_item setPosASL _position;
+
+["hideObjectGlobal", [_item, false]] call EFUNC(common,serverEvent);
+
+// open parachute and ir light effect
+[{
+    params ["_item"];
+
+    if (isNull _item || {getPos _item select 2 < 1}) exitWith {};
```

Comment:
Yes. I don't want to open a parachute if the crate already landed, e.g. when droping from a very low altitude.